src/writeData/components/WriteDataSections.tsx

```diff
@@ -40,7 +43,11 @@ const WriteDataSections: FC = () => {
   return (
     <>
       <FileUploadSection />
-      <ClientLibrarySection />
+      {useSelector(isOrgIOx) ? (
+        <ClientLibrarySectionSql />
+      ) : (
+        <ClientLibrarySection />
+      )}
```

Comment:
This is failing lint checks. Easiest fix is to move the `useSelector` call higher up in the component (say around line 27/28) and store the result in a variable, then refer to the variable. This is one of those "wow, hooks are really stupid sometimes" moments.

```ts
const {searchTerm} = useContext(WriteDataSearchContext)
const isIOx = useSelector(isOrgIOx)

// other code

{isIOx ? (
  <ClientLibrarySectionSql />
  ) : (
    <ClientLibrarySection />
)}